omod/src/main/webapp/pages/conditionlist/manageCondition.gsp

```diff
@@ -70,17 +71,18 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                         datePickerFormat : "yyyy-MM-dd",
                         label            : ui.message('coreapps.stopDate.label'),
                         useTime          : false,
-                        endDate          : new Date()
+                        endDate          : new Date(),
+                        clearButton      : true
```

Comment:
The datetimepicker in uicommons has a boolean configuration property to indicate whether a clear button should be included in the widget.  This ended up being all that was needed to fix RA-2012